### DISCUSSION TITLE
Error fetching Ammonite

### DISCUSSION THREAD

**Discussion by danbills** [scalameta/nvim-metals#281] [Q&A] [answered]
I'm new to this plugin.  I've opened an Ammonite  `.sc` script and see this message

`LSP[metals] Error fetching Ammonite 2.3.8-124-2da846d2 for scala 2.13.7`

Any idea how I could address?

btw Thanks for this awesome project!

**Comment by ckipp01** [comment, marked as answer]
Hey @danbills! So ammonite is a finicky beast in Metals. So what's happening is that when you open an Ammonite script it will default to trying to fetch the latest for your scala version and ammonite version that is the default. Since the default version of Ammonite that we support wasn't (at the time) published for Scala 2.13.7, you end up getting issues like this since it doesn't actually exist. There are a couple ways around this. Under the hood we use https://github.com/alexarchambault/ammonite-runner which allows you to do an extra couple things with comments. For example you put a comment at the top of your file like this:

```
// scala 2.13.6
```
and that will force that version, which should work. There is a blurb about this on the website here: https://scalameta.org/metals/docs/troubleshooting/faq#how-do-i-use-scala-2xx-for-my-script

Or you could find the version of ammonite that is published for 2.13.7 and use that as well.
